openmls/src/group/tests/test_external_commit_validation.rs

```diff
@@ -407,7 +408,7 @@ fn test_valsem243(ciphersuite: &'static Ciphersuite, backend: &impl OpenMlsCrypt
     let proposal_store = ProposalStore::new();
     let (_bob_group, message) = MlsGroup::join_by_external_commit(
         backend,
-        Some(&tree_option),
+        Some(&tree_option), // Note that this isn't actually used.
```

Comment:
Nope, it still uses a code path that ensures that the extension is used. It's not really testing it but since there's no test excercising this afaik I keep this.